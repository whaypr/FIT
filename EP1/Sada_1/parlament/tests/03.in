1
5 4
1 2 3 4
5 6 7 8
9 10 11 12
13 14 15 16
17 18 19 20
3
1 1 5 4
1 2 5 2
4 2 4 4