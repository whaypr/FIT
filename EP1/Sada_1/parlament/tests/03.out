Absolutni hodnota pohodlnosti je 210 bodu.
Absolutni hodnota pohodlnosti je 50 bodu.
Absolutni hodnota pohodlnosti je 45 bodu.
